### PR TITLE
add --watch to copy-modules

### PR DESCRIPTION
**Pull request by jasonLaster**
### Summary of Changes

- adds the ability to watch files so that we can incrementally transpile changed files while testing...
- adds a new `yarn watch` command that replaces `copy-assets-watch`

### CODE REVIEW

**Comment by darkwing** [comment]
Best way to test?

**Comment by jasonLaster** [comment]
The best way to test is to 
1. pull it down
2. run `yarn watch`
3. run `yarn mochih dbg-exceptions` and change the test and a file it uses

File: bin/copy-modules.js [hunk 1]
```diff
@@ -44,13 +53,19 @@ function transformSingleFile(filePath) {
   return out.code;
 }
 
+function transpileFile(file) {
+  if (ignoreFile(file)) {
+    return;
+  }
+
+  const filePath = path.join(__dirname, "..", file);
+  const code = transformSingleFile(filePath);
```
Comment (juliandescottes) [inline bin/copy-modules.js]: we could inline transformSingleFile in here, no need for two separate methods.

File: bin/copy-modules.js [hunk 2]
```diff
@@ -18,13 +20,20 @@ const envConfig = getConfig();
 feature.setConfig(envConfig);
 
 const args = minimist(process.argv.slice(1), {
```
Comment (juliandescottes) [inline bin/copy-modules.js]: Nothing to do with your change but I remember changing the slice(2) (from copy-assets) to slice(1) and forgetting to check the impact. Do you think this should be slice(2)?

Comment (jasonLaster) [reply bin/copy-modules.js]: should be fine...

**Comment by jasonLaster** [comment]
If any issues come up we can easily tweak

**Pull request merged**